code/datums/diseases/advance/symptoms/skin.dm

```diff
@@ -333,3 +333,209 @@ BONUS
 		icon_state = "fleshegg-sneaky"
 	if(LAZYLEN(diseases))
 		AddComponent(/datum/component/infective, diseases)
+
+/*
+//////////////////////////////////////
+Spiked Skin
+	Noticeable.
+	Raises resistance
+	Decreases stage speed
+	No transmission bonus
+
+Thresholds
+  transmission 6 Can give minor armour
+  resistance 6 Does more damage
+
+//////////////////////////////////////
+*/
+/datum/symptom/spiked
+	name = "Cornu Cutaneum"
+	desc = "The virus causes the host to unpredictably grow and shed sharp spines, damaging those near them."
+	stealth = -3
+	resistance = 3
+	stage_speed = -3
+	transmission = 0
+	level = 0
+	symptom_delay_min = 1
+	symptom_delay_max = 1
+	severity = 1
+	base_message_chance = 5
+	prefixes = list("Thorny ", "Horned ")
+	bodies = list("Horn", "Spiked")
+	var/Power = 1
```

Comment:
Why the separate Power var?

Reply:
uhhhh because i didnt code this symptom originally, thats stupid and ill fix it